src/CMakeLists.txt

```diff
@@ -12,3 +12,6 @@ rock_executable(dvl_teledyne_read
 rock_executable(dvl_teledyne_configure
     MainConfigure.cpp
     DEPS dvl_teledyne)
+rock_executable(dvl_teledyne_config_configure
```

Comment:
Since this applies the default config, lets name it `dvl_teledyne_apply_default_config`?

Reply:
Sounds good!

Reply:
I also changed the name of the source file to MainApplyDefaultConfig